src/api/custom-resolvers/mutations/update-loss-payee-financial-details-uk/index.ts

```diff
@@ -0,0 +1,55 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import encrypt from '../../../helpers/encrypt';
+import { ApplicationLossPayeeFinancialUk, SuccessResponse } from '../../../types';
+
+/**
+ * updateLossPayeeFinancialDetailsUk
+ * encrypts sortCode and accountNumber
+ * saves sortCode, sortCodeVector, accountNumber, accountNumberVector, bankAddress to db
+ * @param {Object} GraphQL root variables
+ * @param {ApplicationLossPayeeFinancialUk} GraphQL variables for the ApplicationLossPayeeFinancialUk mutation
+ * @param {Object} KeystoneJS context API
+ * @returns {Object} Object with success flag
+ */
+const updateLossPayeeFinancialDetailsUk = async (root: any, variables: ApplicationLossPayeeFinancialUk, context: Context): Promise<SuccessResponse> => {
+  try {
+    console.info('Updating loss payee financial details UK %s', variables.id);
+
+    const { id, accountNumber, sortCode, bankAddress } = variables;
+
+    /**
+     * object with encrypted data and initialisation vectors
+     * encrypts accountNumber and sortCode
+     * adds the initialisation vectors
+     */
+    const updateData = {
+      accountNumber: encrypt(accountNumber).value,
+      accountNumberVector: encrypt(accountNumber).iv,
+      sortCode: encrypt(sortCode).value,
+      sortCodeVector: encrypt(sortCode).iv,
+      bankAddress,
+    };
+
+    const response = await context.db.LossPayeeFinancialUk.updateOne({
+      where: {
+        id,
+      },
+      data: updateData,
+    });
+
+    if (response) {
+      return {
+        success: true,
```

Comment:
Can this be following?

```javascript
return {
success: Boolean(response),
};
```

Reply:
@ttbarnes should we change to this? everywhere else is consistent with success: true

Reply:
Can we handle this separately please - we're using this pattern everywhere, if we want to change it, we should change all.

What's the rationale for the suggestion @abhi-markan ? I think what we have in place already ( `return { success: true };` ) is simple and easy to read, I don't see a need to pass the response into `Boolean`, which will only return the same thing.

Reply:
This is to keep the code consistent in my opinion, however happy for this to be looked in as a separate PR.